### ISSUE TITLE
Monitor remote certificate via URL

### ISSUE THREAD

**Issue by gueuselambix** [matteocorti/check_ssl_cert#349]
It would be nice if our nagios could monitor a certificate offered by a URL, instead on the local file path.

e.g. ./check_ssl_cert --uri http://foo.bar/cert.pem

**Comment by matteocorti** [comment]
It's a nice idea but it could be better if the --file option would support a URI (e.g. a path or a URI with file:/// http:/// ftp:/// whatever)

I'll take a look

**Comment by matteocorti** [comment]
Feature implemented in 329e28f (release 2.19.0). You can now use any curl supported schemes and submit the URI as a parameter to ```--file```.

E.g.

```
./check_ssl_cert --file file:///some/path/to/cert.crt
./check_ssl_cert --file https://www.example.org/cert.crt
./check_ssl_cert --file ftp://ftp.example.org/cert.crt
```

**Comment by gueuselambix** [comment]
Thank you for this very fast implementation!
